WebRoot/js/zimbraMail/prefs/view/ZmPreferencesPage.js

```diff
@@ -674,6 +674,9 @@ function(parentIdOrElem, text, width, listener, id) {
 	button.setText(text);
 	button.addSelectionListener(listener);
 	this._replaceControlElement(parentIdOrElem, button);
+	if(className) {
```

Comment:
addClassName internally calls DwtControl.prototype.addClassName --> Dwt.addClass  --> Dwt.delClass.
Now, there in Dwt.delClass, the code
`if (!del && !add) { return; }` makes sure that if the className param which is passed in addClassName function is empty, it will not add any class.
So, above if is just an extra and unnecessary check and can be removed.